elastic/kibana
# [Security Solutions][Telemetry] Add first EBT tracking to Entity Analytics page

EPIC issue: https://github.com/elastic/kibana/issues/145276
## Summary

Add 3 telemetry events to the entity risk score feature.

1. Investigate alerts  `Entity Alerts Clicked`
<img width="1498" alt="Screenshot 2023-03-06 at 13 41 14" src="https://user-images.githubusercontent.com/1490444/223113325-5b9cd541-559c-4a3c-bef7-e334825793e0.png">

Event: 
```json
{
    "timestamp": "2023-03-20T14:23:53.452Z",
    "event_type": "Entity Alerts Clicked",
    "context": {
        ...
    },
    "properties": {
        "entity": "host"
    }
}
```
It is triggered on click.
<hr />

2. Entity details clicked `Entity Details Clicked`
<img width="1493" alt="Screenshot 2023-03-06 at 13 37 44" src="https://user-images.githubusercontent.com/1490444/223113002-aaa8d4aa-afd9-4d39-a37e-15496a6ea8c4.png">

Event:  
```json
{
    "timestamp": "2023-03-20T14:26:26.512Z",
    "event_type": "Entity Details Clicked",
    "context": {
        ...
    },
    "properties": {
        "entity": "host"
    }
}
```
It is triggered on click.
<hr />

3. Entity risk filtered `Entity Risk Filtered`
<img width="1498" alt="Screenshot 2023-03-06 at 13 46 40" src="https://user-images.githubusercontent.com/1490444/223114516-91930dff-63e0-447c-9776-b840e47e69bf.png">


Event: 
```json
{
    "timestamp": "2023-03-20T14:31:15.015Z",
    "event_type": "Entity Risk Filtered",
    "context": {
        ...
    },
    "properties": {
        "entity": "user",
        "selectedSeverity": "Critical"
    }
}
```
It is triggered when filtering in by a classification (not triggered when filtering out).


### How to test it?

Open the development console and look for a message like this:
<img width="669" alt="Screenshot 2023-03-20 at 15 34 54" src="https://user-images.githubusercontent.com/1490444/226373050-6e63795e-2f38-4b83-9acc-d5a670793345.png">
You can also check if the events are present in the telemetry server. But they could take a long time to show there. Events triggered during development appear here: http://telemetry-v2-staging.elastic.dev when `telemetry.optIn: true` is enabled inside `kibana.yml`.


Dashboards: https://telemetry-v2-staging.elastic.dev/s/securitysolution/app/dashboards#/view/d47a90f0-a6e4-11ed-a6e6-d32d2209b7b7?_g=(filters%3A!()%2CrefreshInterval%3A(pause%3A!t%2Cvalue%3A0)%2Ctime%3A(from%3Anow-7d%2Fd%2Cto%3Anow))

<img width="1768" alt="Screenshot 2023-03-21 at 10 38 21" src="https://user-images.githubusercontent.com/1490444/226567698-b16092eb-4b6f-4ca1-9a89-580cbcafbcc0.png">


### Checklist

Delete any items that are not applicable to this PR.
